components/esp_lvgl_port/include/esp_lvgl_port_lv_blend.h

```diff
@@ -55,6 +55,21 @@ typedef struct {
 
 extern int lv_color_blend_to_argb8888_esp(asm_dsc_t *asm_dsc);
 
+// Just for compatibility with v9.2.0
```

Comment:
I would not recommend this approach of solving compatibility issues with LVGL 9.2. 
I tried to build lvlgl_port test app on your branch, with the HW accelerating enabled and the test app would not build, neither with LVGL9.1, nor with LVGL9.2.

`peter@peter ➜  ~/esp/esp-bsp/components/esp_lvgl_port/test_apps/lvgl_port git:(pr/409) ✗ idf.py -DSDKCONFIG_DEFAULTS='sdkconfig.defaults;sdkconfig.ci.asm_render' set-target esp32s3 build`

Is your feature working with LVGL9.1? If yes, could you consider using just LVGL9.1 in this MR?

We have agreed to make the HW acceleration support working with LVGL9.1, for now [here](https://github.com/espressif/esp-bsp/pull/357#issuecomment-2356526370), until we figure out what to do with this brekaing changes which was introduced in LVGL9.2.